uwblueprint/childrens-aid-society
# juthika/intake - match referral details w hifi designs

## Notion ticket link

[Convert case referral page to hifi](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* modified placeholders and styling to match design



## Steps to test
1. spin up docker
2. go to localhost:3000/intake and check whether the designs match 
![Screen Shot 2022-09-27 at 8 59 03 PM](https://user-images.githubusercontent.com/77185514/192663989-8de34b86-7349-4e13-a699-0a5908684b4b.png)



## What should reviewers focus on?
* check whether designs match other than the select component - that's being worked on in a different ticket)


## Checklist
